pkgs/development/libraries/eclib/default.nix

```diff
@@ -27,14 +27,6 @@ stdenv.mkDerivation rec {
     rev = "v${version}";
     sha256 = "1kmwpw971sipb4499c9b35q5pz6sms5qndqrvq7396d8hhwjg1i2";
```

Comment:
You need to update the hash as well

Reply:
Wow, good thing I did a PR instead of directly pushing. We really should somehow incorporate the arguments in the hash so that just changing the version fails somehow. Thanks for catching that!